scripts/ci/resolve_ec2_target.sh

```
@@ -0,0 +1,211 @@
+      --query 'Reservations[0].Instances[0].[InstanceId,State.Name,PublicIpAddress,PrivateIpAddress,PublicDnsName,join(`,`,SecurityGroups[].GroupId)]' \
+      --output text 2>/tmp/ec2-describe-error.log
+    return
+  fi
+
+  if [ -n "${EC2_INSTANCE_NAME}" ]; then
+    aws ec2 describe-instances \
+      --region "${AWS_REGION}" \
+      --filters \
+        "Name=tag:Name,Values=${EC2_INSTANCE_NAME}" \
+        "Name=instance-state-name,Values=pending,running,stopping,stopped" \
+      --query 'Reservations[0].Instances[0].[InstanceId,State.Name,PublicIpAddress,PrivateIpAddress,PublicDnsName,join(`,`,SecurityGroups[].GroupId)]' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)